main.tf

```diff
@@ -80,3 +89,17 @@ resource "google_project_iam_member" "pubsub_sa_create_oidc_token" {
   member     = "serviceAccount:${local.google_pubsub_sa_email}"
   depends_on = [google_project_service.pubsub_api]
 }
+
+resource "google_project_iam_member" "pubsub_sa_acknowledge_for_dead_letter" {
+  # See https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/pubsub_subscription#dead_letter_policy
```

Comment:
I think we should put these comments out of the block at the top of the block instead.
```
# See https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/pubsub_subscription#dead_letter_policy
resource "google_project_iam_member" "pubsub_sa_acknowledge_for_dead_letter"
```

Reply:
I don't think the comment is required, as it doesn't add any value.

Reply:
The comment adds value in the sense, when we want to debug something, it will give us a clue why we required providing this role. So having the source of info helps.

Reply:
I agree that if you have the sources it will be easy for you to look into it. 
But actually, the source isn't something hard to find it.  
if you want to find the details of the resource what you will do? normally without this comment will be easy to find, isn't it?

Reply:
Hello @ahmedzidan, Can have a look [here](https://github.com/airasia/terraform-google-kms_keyring/pull/1#discussion_r538993730) for similar response